.github/workflows/make_bundle_conda.yml

```diff
@@ -79,7 +88,8 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          repository: conda-forge/napari-feedstock
+          repository: jaimergp/napari-feedstock  # REVERT to conda-forge
```

Comment:
Is this a TODO?

Reply:
I think that it is waiting on https://github.com/conda-forge/napari-feedstock/pull/87